lib/board.rb

```diff
@@ -26,4 +26,19 @@ def invert_board
       column_index -= 1
     end
   end
+  
+  def print_board
+    puts game_board.keys.join
+    @horizontal_board.values.each do |row|
+      subbed_row = []
+      row.each do |slot|
+        if slot == nil
```

Comment:
glad this conditional worked... why did it take us so long to think of this simple option again? lol